storage/connectors/bigquery/src/main/java/feast/storage/connectors/bigquery/writer/BigQuerySinkHelpers.java

```diff
@@ -17,34 +17,30 @@
 package feast.storage.connectors.bigquery.writer;
 
 import com.google.api.services.bigquery.model.TimePartitioning;
-import feast.proto.types.FeatureRowProto.FeatureRow;
+import com.google.cloud.bigquery.DatasetId;
 import org.apache.beam.sdk.io.gcp.bigquery.TableDestination;
-import org.apache.beam.sdk.transforms.SerializableFunction;
-import org.apache.beam.sdk.values.ValueInSingleWindow;
 
-public class GetTableDestination
-    implements SerializableFunction<ValueInSingleWindow<FeatureRow>, TableDestination> {
+public class BigQuerySinkHelpers {
+  public static TableDestination getTableDestination(DatasetId dataset, String featureSetKey) {
+    String[] splitName = featureSetKey.split("/");
 
-  private String projectId;
-  private String datasetId;
-
-  public GetTableDestination(String projectId, String datasetId) {
-    this.projectId = projectId;
-    this.datasetId = datasetId;
-  }
-
-  @Override
-  public TableDestination apply(ValueInSingleWindow<FeatureRow> input) {
-    String[] splitName = input.getValue().getFeatureSet().split("/");
+    if (splitName.length != 2) {
+      throw new RuntimeException("FeatureRow key is invalid. Must be project/feature-set");
```

Comment:
If the `project` is not set then it should be configured to write to the `default` project. Meaning `project` is optional.